src/vdaf/prio3.rs

```diff
@@ -350,10 +352,11 @@ where
 {
     /// Construct an instance of this Prio3 VDAF with the given number of aggregators and the
     /// underlying type.
-    pub fn new(num_aggregators: u8, typ: T) -> Result<Self, VdafError> {
+    pub fn new(num_aggregators: u8, num_proofs: u8, typ: T) -> Result<Self, VdafError> {
         check_num_aggregators(num_aggregators)?;
         Ok(Self {
             num_aggregators,
+            num_proofs,
             typ,
             phantom: PhantomData,
         })
```

Comment:
This makes it possible to create a Prio3 instantiation with `num_proofs = 0`.

Interestingly it turns out to work, i.e. no panics, evaluates the function correctly but with no proofs nor proof validation. But it leaves open the footgun of a user instantiating a "DAF" instead of a "VDAF".